packages/wp-story-editor/src/components/documentPane/constants.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const WP_DOCUMENT_PANEL_NAMES = {
+  PUBLISH: 'publishing',
+  STATUS: 'status',
```

Comment:
part 2 of my outstanding question - so I'm defining `STATUS` here, do we need to pass this through some how to prevent a separate string that maybe doesn't match with another CMS's name for this panel? will it always be status??